.husky/pre-commit

```diff
@@ -5,11 +5,20 @@
 export FORCE_COLOR=1
 
 npm run lint
-npm run unit -- --onlyChanged
 
+# Jest unit tests
+npm run test
+
+# Update version in package.json to match VERSION file
 npm run sync:version
 git add package.json
 
+# Include updated caniuse-lite database used by browserslist
+npx browserslist@latest --update-db
+git add package.json
+git add package-lock.json
```

Comment:
Is it a problem to add package.json and package-lock.json here? Could see this being an issue if somebody has messed with their files for other reasons.

Reply:
Our pre-commit hook is starting to get pretty cumbersome.  Future effort to speed that up